packages/material-ui/src/ArrayFieldTemplate/ArrayFieldTemplate.tsx

```diff
@@ -62,16 +50,16 @@ const ArrayFieldDescription = ({
 
 // Used in the two templates
 const DefaultArrayItem = (props: any) => {
-  const { ArrowDownwardIcon, ArrowUpwardIcon, Box, Grid, IconButton, Paper, RemoveIcon } = useContext(MuiComponentContext);
+  const { ArrowDownwardIcon, ArrowUpwardIcon, Box, Grid, IconButton, Paper, RemoveIcon } = useMuiComponent();
   const btnStyle = {
     flex: 1,
     paddingLeft: 6,
     paddingRight: 6,
     fontWeight: 'bold',
-    minWidth: 0
+    minWidth: 0,
   };
   return (
-    <Grid container={true} key={props.key} alignItems="center">
+    <Grid container={true} key={props.key} alignItems='center'>
```

Comment:
Can you revert **all** of the double-to-single quote changes on the props passed to components? I'm guessing that your IDE auto-fix is on and you have them set to single quotes. It breaks from the patterning in the rest of the library